lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.h

```diff
@@ -43,11 +43,6 @@
 #include <ufe/path.h>
 #endif
 
-// Conditional compilation due to Maya API gap.
-#if MAYA_API_VERSION >= 20200000
-#define MAYA_ENABLE_UPDATE_FOR_SELECTION
-#endif
```

Comment:
This define is gone (since we are always > 2020 now) so I removed the ifdef checks related to it (above).